File: Aliyunplayer/player_demo/src/main/java/com/aliyun/vodplayerview/fragment/BaseFragment.java
package com.aliyun.vodplayerview.fragment;

import androidx.fragment.app.Fragment;

public abstract class BaseFragment extends Fragment {

    /**
     * 默认配置
     */
    public abstract void defaultPlayInfo();

    /**
     * 使用此配置
     */
    public abstract void confirmPlayInfo();
}

File: app/src/main/java/com/example/blogsystem/base/BaseFragment.kt
package com.example.blogsystem.base

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

open class BaseFragment : Fragment, Init {

    protected val TAG: String = javaClass.simpleName
    protected lateinit var mRootView: View

    constructor() : super()
    constructor(contentLayoutId: Int) : super(contentLayoutId)

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        mRootView = super.onCreateView(inflater, container, savedInstanceState)!!
        return mRootView
    }
}
